pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -1320,7 +1320,7 @@ func Convert_v1_VirtualMachine_To_api_Domain(vmi *v1.VirtualMachineInstance, dom
 		domain.Spec.Devices.Controllers = append(domain.Spec.Devices.Controllers, api.Controller{
 			Type:  "scsi",
 			Index: "0",
-			Model: "virtio-scsi",
+			Model: translateModel(c, "virtio-scsi"),
```

Comment:
I have a local version of this patch where I just call

```go
translateModel(c, "virtio")
```

and don't touch the `translateModel()` function at all. Any reason to special-case virtio-scsi here?

Reply:
Good point. I kind  of wanted to keep the connection to the special value, but really not needed. Will change it to the way you suggest.

Reply:
Done.